test/swoosh/test_assertions_test.exs

```diff
@@ -4,87 +4,107 @@ defmodule Swoosh.TestAssertionsTest do
+      |> deliver()
 
-    Swoosh.Adapters.Test.deliver(email, nil)
     {:ok, email: email}
   end
 
+  test "assert any email sent" do
+    assert_email_sent()
+  end
+
+  test "assert any email sent with no emails sent" do
+    receive do
+      _ -> nil
+    end
+
+    assert_raise ExUnit.AssertionError, fn ->
+      assert_email_sent()
+    end
+  end
+
   test "assert email sent with correct email", %{email: email} do
-    assert_email_sent email
+    assert_email_sent(email)
   end
 
   test "assert email sent with some content matched by a regex" do
-    assert_email_sent text_body: ~r/some text/, html_body: ~r/html$/
+    assert_email_sent(text_body: ~r/some text/, html_body: ~r/html$/)
   end
 
   test "assert email sent with specific params" do
-    assert_email_sent [subject: "Hello, Avengers!", to: "steve.rogers@example.com"]
+    assert_email_sent(subject: "Hello, Avengers!", to: "steve.rogers@example.com")
   end
 
   test "assert email sent with specific to (list)" do
-    assert_email_sent [to: ["steve.rogers@example.com"]]
+    assert_email_sent(to: ["steve.rogers@example.com", "bruce.banner@example.com"])
   end
 
   test "assert email sent with wrong subject" do
     assert_raise ExUnit.AssertionError, fn ->
-      assert_email_sent [subject: "Hello, X-Men!"]
+      assert_email_sent(subject: "Hello, X-Men!")
     end
   end
 
   test "assert email sent with wrong from" do
     assert_raise ExUnit.AssertionError, fn ->
-      assert_email_sent [from: "thor.odinson@example.com"]
+      assert_email_sent(from: "thor.odinson@example.com")
     end
   end
 
   test "assert email sent with wrong to" do
     assert_raise ExUnit.AssertionError, fn ->
-      assert_email_sent [to: "bruce.banner@example.com"]
+      assert_email_sent(to: "loki.odinson@example.com")
     end
   end
 
   test "assert email sent with wrong to (list)" do
     assert_raise ExUnit.AssertionError, fn ->
-      assert_email_sent [to: ["bruce.banner@example.com"]]
+      assert_email_sent(to: ["steve.rogers@example.com"])
     end
   end
 
   test "assert email sent with wrong cc" do
     assert_raise ExUnit.AssertionError, fn ->
-      assert_email_sent [cc: "bruce.banner@example.com"]
+      assert_email_sent(cc: "bruce.banner@example.com")
     end
   end
 
   test "assert email sent with wrong bcc" do
     assert_raise ExUnit.AssertionError, fn ->
-      assert_email_sent [bcc: "bruce.banner@example.com"]
+      assert_email_sent(bcc: "bruce.banner@example.com")
     end
   end
 
   test "assert email sent with wrong email", %{email: email} do
     wrong_email = new() |> subject("Wrong, Avengers!")
 
     message =
-      String.trim(
-        """
-        No message matching {:email, ^email} after 0ms.
-        The following variables were pinned:
-          email = #{inspect(wrong_email)}
-        Process mailbox:
-          {:email, #{inspect(email)}}
-        """
-      )
+      String.trim("""
+      No message matching {:email, ^email} after 0ms.
```

Comment:
is this change from formatter?

Reply:
yep, I have my editor to auto format on save with `mix format`, all formatting changes are from `mix format`.